src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  const sum = days * 40;
+
+  if (days < 3) {
+    return sum;
+  }
+
+  if (days >= 7) {
+    return sum - 50;
+  }
+
+  if (days >= 3) {
+    return sum - 20;
```

Comment:
Use a constant for the number of days that qualifies for a short term rental, and another constant for the corresponding discount amount, instead of using literals '3' and '20'.